remindme-notifier/src/main/java/it/andreafailli/remindme/notifier/controllers/CronHandlerController.java

```diff
@@ -0,0 +1,34 @@
+package it.andreafailli.remindme.notifier.controllers;
+
+import org.slf4j.ext.XLogger;
+import org.slf4j.ext.XLoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import it.andreafailli.remindme.common.services.ReminderService;
+
+@RestController
+@RequestMapping(CronHandlerController.BASE_URL)
+public class CronHandlerController {
+	
+public static final String BASE_URL = "/";
+	
+	private static final XLogger LOGGER = XLoggerFactory.getXLogger(CronHandlerController.class);
+	
+	@Autowired
+	private ReminderService reminderService;
+
+	@RequestMapping
+	public ResponseEntity<Void> handler() {
+		LOGGER.entry();
+		
+		//TODO: list current reminders and send notifications
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)